dagmc_stats.py

```diff
@@ -201,3 +201,43 @@ def get_triangle_aspect_ratio(my_core, meshset, geom_dim):
         t_a_r.append(top/bottom)
 
     return t_a_r
+
+def get_area_triangle(my_core, meshset):	
+    """
+    Gets the triangle area (according to the equation: sqrt(s(s - a)(s - b)(s - c)), where s = (a + b + c)/2)
+    
+    inputs
+    ------
+    my_core : a MOAB Core instance
+    meshset : a meshset containing a certain part of the mesh
+    
+    outputs
+    -------
+    area : (list) the triangle areas for all triangles in the meshset
+    """
+    
+    area = []
+    tris = my_core.get_entities_by_type(meshset, types.MBTRI)
+
+    for triangle in tris:
```

Comment:
Not a big deal but I would use more consistent naming convention:
`verts`/`vert`
`tris`/`tri` or `triangles`/`triangle`